src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
+    private int supplyInt = 0;
+    private int buyInt = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] textArray = fileTextToArray(fromFileName);
+        countSupplyAndBuy(textArray);
+        writeTextToFile(toFileName);
+    }
+
+    public String[] fileTextToArray(String fromFileName) {
```

Comment:
```suggestion
   private String[] fileTextToArray(String fromFileName) {
```

these methods are helper ones, so ve can make them private